imports.go

```diff
@@ -120,8 +120,18 @@ type Imports []*Import
 
 // Validate checks if an import is valid for the wrapping account
 func (i *Imports) Validate(acctPubKey string, vr *ValidationResults) {
+	m := make(map[Subject]string)
 	for _, v := range *i {
 		v.Validate(acctPubKey, vr)
+		// if it as service, insure that the subject the client is making the request is unique
+		if v.IsService() {
+			c := m[v.Subject]
```

Comment:
Would be more:
```
if c, exist := m[v.Subject]; exist {
  vr.AddError("account already imports a service at %s from account %s", v.Subject, c)
} else {
  m[v.Subject] = v.Account
}
```

Reply:
just providing a message so that it can be identified, if the account was the same, that would have been a different validation error which would have surfaced earlier.